src/include/execution/vm/bytecode_handlers.h

```diff
@@ -1300,11 +1336,13 @@ VM_OP_WARM void OpPow(terrier::execution::sql::Real *result, const terrier::exec
 // ---------------------------------------------------------
 
 VM_OP_WARM void OpValIsNull(terrier::execution::sql::BoolVal *result, const terrier::execution::sql::Val *val) {
-  terrier::execution::sql::IsNullPredicate::IsNull(result, *val);
+  bool is_null = terrier::execution::sql::IsNullPredicate::IsNull(*val);
```

Comment:
Modify this to operate on primitive booleans and use these two bytecodes instead of adding new ones.